test/src/e2e_vm_tests/test_programs/should_pass/language/fallback_only/src/main.sw

```diff
@@ -2,13 +2,14 @@ contract;
 
 use std::execution::run_external;
 
-#[namespace(SRC1822)]
 storage {
-    target: ContractId = ContractId::zero(),
+    SRC1822 {
+        target: ContractId = ContractId::zero(),
+    }
 }
 
 #[fallback]
 #[storage(read)]
 fn fallback() {
-    run_external(storage.target.read())
+    run_external(storage::SRC1822.target.read())
```

Comment:
Since this is a breaking change regardless, we may indeed want to make the namespace annotation use the new separators, even if it's not going to be long for this world.